app/src/main/java/com/example/groceriesmanager/Models/Recipe.java

```diff
@@ -45,44 +61,73 @@ public Recipe(JSONObject jsonObject) throws JSONException {
     }
 
     public String getImage_url() {
```

Comment:
nit: Naming style of function name getImage_url() should be changed to -> getImageURL | getImageUrl to be consistent with other functions and remove the '_'.